source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -18,123 +18,147 @@
-                <!-- ko foreach: values -->
-                <td data-bind="css: { 'exceeds-expectations': showValue() && value() > 3 && value() <= 4 , 'poor': showValue() && value() < 3 && value() >= 1}">
-                    <span data-bind="if: !editable"><span data-bind="text: value, visible: showValue()"></span></span>
-                    <span data-bind="if: editable == true"><input class="calification-input" data-bind="value: value, css: { invalid: !isValid()}, event: { focusin: onFocus, blur: onBlur} " /></span>
-                </td>
-                <!-- /ko -->
-            </tr>
-            <!-- /ko -->
-        </tbody>
-    </table>
-    <div class="footer-evaluation" data-bind="if: isEvaluationEditable()">
-        <button class="button-evaluations primary-btn finish-btn" data-bind="click: onFinish">Finalizar</button>
-        <p class="explanatory-text">Recuerda que una vez finalizada tu evaluación no podrás volver a editarla.</p>
+    <h4>Comentarios de la Evaluación</h4>
+    <div class="comments-container comments-company-container">
+      <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
     </div>
-    <div id="loader-id" class="loader-container" data-bind="visible: isLoading()">
-        <div class="loader"></div>
+    <h4>Comentarios del Evaluado</h4>
+    <div class="comments-container comments-company-container">
+      <textarea class="content-editable big-comment" data-bind="value: evaluatedComment, enable: evaluation.readyForDevolution && !evaluation.finished && userView == 3"></textarea>
     </div>
-    <script type="text/javascript">
-        var calificationPeriod = "@ViewBag.Period";
-        var calificationUserName = "@ViewBag.UserName";
-    </script>
+    <!-- /ko -->
+    <!--ko if: userView != 3 && !evaluation.readyForDevolution -->
+    <div class="top-right">
+      <h4>Comentarios Generales</h4>
+      <div class="comments-container">
+        <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
+      </div>
+    </div>
+    <!-- /ko -->
+  </div>
+  <table data-bind="attr: {class: numberOfColumns}, css: {table: true, 'calification-items': true}">
+    <thead id="header-fixed">
+      <tr>
+        <th class="calificators"><h1>Evaluadores</h1></th>
+        <!-- ko foreach: califications -->
+        <th class="calificators">
+          <span data-bind="if: hasShowIcon"><i data-tips="" data-bind="css: { icon: true, 'top-tip': true, 'hide-column': !show()}, click: $root.toggleVisibilityColumn"></i></span>
+          <span data-bind="text: evaluatorEmployee.replace('_company', 'Empresa')"></span>
+        </th>
+        <!-- /ko -->
+        <th class="calificators">
+          <p>BUTTON HERE</p>
```

Comment:
could you add at least an unicode triangle or arrow or any other symbol? 

For example: http://unicode-table.com/es/25BC/

(Remember that anything that we merge should be production ready)